saleor/app/validators.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from django.core.validators import URLValidator
+from django.utils.regex_helper import _lazy_re_compile  # type: ignore
+
+
+class AppURLValidator(URLValidator):
+    validator = URLValidator
+    host_re = "(" + validator.hostname_re + validator.domain_re + "|localhost)"
+    regex = _lazy_re_compile(
+        r"^(?:[a-z0-9.+-]*)://"  # scheme is validated separately
```

Comment:
Could we add the reason why the `scheme is validated separately` in this comment?

Reply:
It's took directly from django, that will be okay if I will add source of this piece of code?